static/src/javascripts/projects/commercial/modules/comment-adverts.spec.ts

```diff
@@ -62,7 +62,7 @@ const refreshAdvert = refreshAdvert_;
 const mockHeight = (height: number) => {
 	// this is an issue with fastdom's typing of measure: () => Promise<void>
 	jest.spyOn(fastdom, 'measure').mockReturnValue(
-		(Promise.resolve(height) as unknown) as Promise<void>,
+		Promise.resolve(height) as unknown as Promise<void>,
```

Comment:
This is obviously wrong, isn’t it?

`Promise<number>` can’t also be `Promise<void>`.